### PR TITLE
Provide a polyfill for IE11+

### PR DESCRIPTION
**Pull request by aarongranick-okta**
Provides an entrypoint that implements all required polyfills. To use it, import or require `@okta/okta-auth-js/polyfill` at or near the beginning of your application's code.

```javascript
import '@okta/okta-auth-js/polyfill';
```

or

```javascript
require('@okta/okta-auth-js/polyfill');

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,15 +42,30 @@ If you run into problems using the SDK, you can:
...
 
-* ES6 Promise
+* ES Promise
 * Array.from
 * TextEncoder
+* Object.assign
+* UInt8 typed array
+* webcrypto (crypto.subtle)
+
+This module provides an entrypoint that implements all required polyfills. To use it, import or require `@okta/okta-auth-js/polyfill` at or near the beginning of your application's code.
```
Comment (shuowu) [inline README.md:58]: Add an option to use it in the `script` tag?

Comment (aarongranick-okta) [reply README.md:58]: That's what I was discussing about "the CDN option"

Comment (aarongranick-okta) [reply README.md:58]: You *could* include the webpack bundle from the node modules directory I suppose

File: packages/okta-auth-js/webpack.polyfill.config.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* global __dirname */
+var path    = require('path');
+var _ = require('lodash');
+var commonConfig = require('./webpack.common.config');
+var webpack = require('webpack');
+var fs      = require('fs');
+
+var license = fs.readFileSync('lib/license-header.txt', 'utf8');
+
+module.exports = _.extend({}, _.cloneDeep(commonConfig), {
```
Comment (shuowu) [inline packages/okta-auth-js/webpack.polyfill.config.js:10]: should we uglify this polyfill?

Comment (aarongranick-okta) [reply packages/okta-auth-js/webpack.polyfill.config.js:10]: meh

File: README.md (hunk 1 above)
Comment (shuowu) [reply README.md:58]: yep, I remember there are plugins for that.

File: packages/okta-auth-js/webpack.polyfill.config.js (hunk 2 above)
Comment (aarongranick-okta) [reply packages/okta-auth-js/webpack.polyfill.config.js:10]: OK I added the uglify

**Pull request merged**